sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -986,6 +993,10 @@ def on_port_update_event(self, port_change_event):
                 self.port_dict[lport]['speed'] = port_change_event.port_dict['speed']
```

Comment:
Can you please add the invalid speed check before copying to local dictionary, this workaround is needed until the original issue is fixed <https://github.com/Azure/sonic-swss/issues/2276 >

if (port_change_event.port_dict['speed']  != N/A):
    self.port_dict[lport]['speed'] = port_change_event.port_dict['speed']

Reply:
tracking the original fix: https://github.com/Azure/sonic-swss/issues/2276

Reply:
added